pkg/lightning/backend/importer.go

```diff
@@ -39,9 +39,14 @@ const (
 )
 
 var (
-	requiredTiDBVersion = *semver.New("2.1.0")
-	requiredPDVersion   = *semver.New("2.1.0")
-	requiredTiKVVersion = *semver.New("2.1.0")
+	// Importer backend is compatible with TiDB [2.1.0, 6.0.0).
+	requiredMinTiDBVersion = *semver.New("2.1.0")
+	requiredMinPDVersion   = *semver.New("2.1.0")
+	requiredMinTiKVVersion = *semver.New("2.1.0")
+	// TODO: bump max versions based on the version define in Makefile.
+	requiredMaxTiDBVersion = *semver.New("6.0.0")
+	requiredMaxPDVersion   = *semver.New("6.0.0")
+	requiredMaxTiKVVersion = *semver.New("6.0.0")
```

Comment:
you could use the [`BumpMajor` function](https://pkg.go.dev/github.com/coreos/go-semver/semver#Version.BumpMajor) 🙃

Reply:
Sure, I plan to implement the TODO in another PR (will send soon after this PR is merged) along with unification of BR and Lightning `LDFLAGS=-Xpkg/.../ReleaseVersion=someversion`.